src/main/java/nl/dtls/fairdatapoint/service/metadatametrics/FairMetadataMetricsServiceImpl.java

```diff
@@ -105,13 +83,12 @@ public void setMetadataMetrics(Map<String, String> metadataMetrics) {
      * We are using this method to reduce the NPath complexity measure. This method add a FM to the
      * list if the metric valueUri URI is provided.
      *
-     * @param metrics List<Mertic> object
-     * @param uri Metric uri
-     * @param typeUri Metric typeUri uri
+     * @param metrics  List<Mertic> object
+     * @param uri      Metric uri
+     * @param typeUri  Metric typeUri uri
```

Comment:
This indentation style is prone to inconsistencies across IDEs. See style discussion.

Reply:
Resolved by https://github.com/FAIRDataTeam/FAIRDataPoint/pull/53#issuecomment-536993575